modules/eco-configuring-machine-health-check-with-poison-pill.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// *nodes/nodes/eco-poison-pill-operator.adoc
+
+[id="configuring-machine-health-check-with-poison-pill_{context}"]
+= Configuring machine health checks to use the Poison Pill Operator 
+
+Use the following procedure to configure the machine health checks to use the Poison Pill Operator as a remediation provider.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a `PoisonPillRemediationTemplate` CR
+
+.. Define the `PoisonPillRemediationTemplate` CR:
++
+[source,yaml]
+----
+apiVersion: poison-pill.medik8s.io/v1alpha1
+kind: PoisonPillRemediationTemplate
+metadata:
+  namespace: openshift-machine-api
+  name: poisonpillremediationtemplate-sample
+spec:
+  template:
+    spec: {}
+----
+
+.. To create the `PoisonPillRemediationTemplate` CR, run the following command:
++
+[source,terminal]
+----
+$ oc create -f <ppr-name>.yaml
```

Comment:
See my later comment about a more common usage in our doc set: `<file-name>.yaml`